docs/docs/rad-lab-ui/ui_installation/infrastructure.md

```diff
@@ -176,19 +143,13 @@ Personal Access Token) to fetch the RAD Lab modules from your repo.
 1. Click on **Generate token**
 1. Copy the value of the token.
 
-You can add a secret directly on the command line using below command (replace `${GIT_PERSONAL_ACCESS_TOKEN}` with the token copied from the Github UI) :
+You can add a secret directly on the command line using below command (replace `${GIT_PERSONAL_ACCESS_TOKEN}` with the token copied from the Github UI), where `${PROJECT_ID}` corresponds to the RAD Lab UI project ID. 
 
 ```bash
 echo -n "${GIT_PERSONAL_ACCESS_TOKEN}" | \
     gcloud secrets versions add $(terraform output -json | jq -r .git_personal_access_token_secret_id.value) --data-file=- --project ${PROJECT_ID}
 ```
 
-To get `PROJECT_ID`, run below command. Copy output and replace `${PROJECT_ID}`
```

Comment:
We can still keep this section as it directly gives the Project ID

Reply:
After running the enablement sessions, it's incredibly confusing for people who go through the instructions, as they think they have to re-run that statement.  That's why I removed it.